scripts/hooks/post-checkout

```diff
@@ -1,2 +1,2 @@
 #!/bin/sh
-npm run npx hereby -- generate-diagnostics
+npx hereby -- generate-diagnostics
```

Comment:
Maybe
```suggestion
npx hereby -- generate-diagnostics || true
```
So the checkout itself always actually works even if the post-checkout fails?

Reply:
Ah, yeah, both of these need that. That and I wanted to avoid `npx` because it has a very very slow startup time compared to what it's running.

Reply:
Updated, and now it uses `node_modules/.bin/...`, which in my testing works fine on Windows (this all is run via git-bash).

Reply:
Like wow

![image](https://github.com/microsoft/TypeScript/assets/5341706/ba06dd8a-a282-4315-a7db-2e8520697286)